configs/application/manifest-local.json

```diff
@@ -27,8 +27,8 @@
 	"splashScreenImage": "http://localhost:3375/assets/img/FinsembleSplash.svg",
 	"appAssets": [
 		{
-			"src": "https://assets.finsemble.com/assimilation/dev/pr/assimilation.zip",
-			"version": "5.0.0.2",
+			"src": "https://assets.finsemble.com/assimilation/assimilation_5.0.1.0.zip",
```

Comment:
Is this correct? I don't know off hand.

Reply:
Yup this is from the A-team and should be patched to 5.0.1.0.